install/installer/pkg/components/image-builder-mk3/networkpolicy.go

```diff
@@ -15,6 +16,34 @@ import (
 
 func networkpolicy(ctx *common.RenderContext) ([]runtime.Object, error) {
 	labels := common.DefaultLabels(Component)
+	var ingressRules []networkingv1.NetworkPolicyPeer
+	// Allow all ingress in workspace clusters
+	// until https://github.com/gitpod-io/ops/issues/6905 is fixed.
+	if ctx.Config.Kind != config.InstallationWorkspace {
```

Comment:
In other words, for workspace installation types, [we'll allow all (by defining no ingress rules)](https://kubernetes.io/docs/concepts/services-networking/network-policies/#allow-all-ingress-traffic)